packages/shared/src/components/modals/OnboardingModal.tsx

```diff
@@ -118,6 +164,19 @@ function OnboardingModal({
     return validations;
   }, [onboardingSteps]);
 
+  useEffect(() => {
+    trackEvent({
+      event_name: AnalyticsEvent.Impression,
+      target_type: TargetType.MyFeedModal,
+      target_id: OnboardingVersion.V2,
```

Comment:
This is not aligned with how we track target_id for the `my feed button` just double sure with @idoshamun what it should be for both.

Reply:
We have two impressions, the button which has `my feed button` and the `my feed modal` which is for when the modals were seen.

Reply:
Most of the `tracking_id` I am actually not 100% on their correctness. I think it would be amazing if we can have a guideline on how to identify it, but if the impact on PR is not too much, then probably it is not necessary.

Reply:
@rebelchris can you elaborate on what's not aligned please? It seems good to me

Reply:
@idoshamun This comment is related to we normally don't track flag values and rely on what get's added on the default tracking object.
But since we now decided to track specific values rather you can ignore this comment.

Just wanted you to note we do have different ways of using "target_id" at this point.
New ones are flags, old ones are actual positions (hardcoded)

Reply:
Thanks for noting it down. As we proceed, I believe it will be more flags with some unique cases that will use placement ids. It really depends on what we want to test at the end of the day